backend/game_server/lib/cell.ex

```diff
@@ -0,0 +1,32 @@
+defmodule BernacleServer.Cell do
+	alias BernacleServer.{Entity,Cell}
+	alias BernacleServer.Helpers.{Physic,Vector}
+
+
+	def born(position, velocity) do
+		Entity.new
```

Comment:
I'm not conviced to this bracket less notation. I mean from readable perspective, when I'm see `Entity.new()` I'm sure this function is really called. When I see `Entity.new` I could think that this is some reference to function that could be called later.